model/field/rum_v3_mapping.go

```diff
@@ -130,3 +130,17 @@ func Mapper(shortFieldNames bool) func(string) string {
 		return s
 	}
 }
+
+func InverseMapper(shortFieldNames bool) func(string) string {
+	m := map[string]string{}
+	for k, v := range rumV3Mapping {
+		m[v] = k
+	}
```

Comment:
The inverse mapping is done every time `InverseMapper` is called, which happens for every sample event. Please make the mapping static instead.